src/components/storage/CockpitStorageIntegration.scss

```diff
@@ -57,3 +60,25 @@ ul.cockpit-storage-integration-requirements-hint-list {
         color: unset;
     }
 }
+
+// Simulate Backdrop behavior for modals in the iframe
+%iframe-zindex {
+  position: relative;
+  z-index: 10;
+}
```

Comment:
This is SCSS that says "here's some reusable SCSS code; assign it a name %iframe-zindex to be included elsewhere"

Reply:
Here's some docs on `@extend`, with the first showing a % block as an extended block:

https://sass-lang.com/guide/#extend-inheritance
https://sass-lang.com/documentation/at-rules/extend/